src/Data.js

```diff
@@ -196,7 +196,7 @@ CM.ConfigData.DragonAuraInfo = {type: 'bool', group: 'Tooltip', label: ['Extra D
 // Statistics
 CM.ConfigData.Stats = {type: 'bool', group: 'Statistics', label: ['Statistics OFF', 'Statistics ON'], desc: 'Extra Cookie Monster statistics!', toggle: true};
 CM.ConfigData.MissingUpgrades = {type: 'bool', group: 'Statistics', label: ['Missing Upgrades OFF', 'Missing Upgrades ON'], desc: 'Shows Missing upgrades in Stats Menu. This feature can be laggy for users with a low amount of unlocked achievements.', toggle: true};
-CM.ConfigData.MissingAchievements = {type: 'bool', group: 'Statistics', label: ['Show Normal Missing Achievements', 'Show Shadow Missing Achievements', 'Show All Missing Achievements', 'Hide Missing Achievements'], desc: 'Shows Missing normal or shadow achievements in Stats Menu.', toggle: false};
+CM.ConfigData.MissingAchievements = {type: 'bool', group: 'Statistics', label: ['Missing Normal Achievements ON', 'Missing Shadow Achievements ON', 'All Missing Achievements ON', 'Missing Achievements OFF'], desc: 'Shows Missing normal or shadow achievements in Stats Menu.', toggle: false};
```

Comment:
Please change it so that `Missing Achievements OFF` is the first option. That why `CM.Options.MissingAchievements == 0` corresponds to the options being off.